src/iop/denoiseprofile.c

```diff
@@ -41,10 +42,15 @@
 #include <xmmintrin.h>
 #endif
 
+// which version of the non-local means code should be used?  0=old (this file), 1=new (src/common/nlmeans_core.c)
+#define USE_NEW_IMPL_CL 0
```

Comment:
@ralfbrown was trying to figure out if there was any dedup opportunities between `nlmeans_denoiseprofile_cl` and `nlmeans_denoise_cl` as they look fairly similar. Then came across this define and wondered if the plan still was to switch to the new code? Not completely clear from the discussion in the PR at the time. Maybe one of the two paths (and then unused code) could be cleaned up?

Reply:
Ideally we would use the new path, but I have no way of testing it (no working OpenCL here, video card is too old) so I never activated it.  If someone can verify that the new version gives the same results, we can switch to it and remove the old version.